docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
So many questions...

Why a composer container? IIRC, composer is a dev dependency, and most have it installed. https://hub.docker.com/r/composer/composer/ itself suggests running it through an alias.

Why not install it in the harvester container itself?

Why not composer/composer:alpine, if we must?